lua/weapons/weapon_spraypaint/cl_init.lua

```diff
@@ -1,66 +1,576 @@
 ﻿-- This file is subject to copyright - contact swampservers@gmail.com for more information.
 -- INSTALL: CINEMA
 include("shared.lua")
-SWEP.Instructions = "Primary: Spray White\nSecondary: Spray Black"
 
-function SWEP:DrawWorldModel()
+function SWEP:ShakeCan()
+    timer.Create("capshakesnd__" .. self:EntIndex(), 0.3, 1, function()
```

Comment:
**timer.Simple !!!!!!!!!!!**

Reply:
and don't forget the IsValid() check

Reply:
i use regular timers because this needs to be interruptible